src/KubernetesClient/LineSeparatedHttpContent.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace k8s
 {
-    internal class LineSeparatedHttpContent : HttpContent
+    internal sealed class LineSeparatedHttpContent : HttpContent
```

Comment:
any reason to make internal class sealed?

Reply:
There's a ticket with some introduction https://github.com/dotnet/runtime/issues/49944